privacy-on-beam/pbeam/aggregations.go

```diff
@@ -664,10 +663,12 @@ func (fn *emitPartitionsNotInTheDataFn) ProcessElement(partitionKey beam.X, valu
 	if err := fn.partitionEnc.Encode(partitionKey, &partitionBuf); err != nil {
 		log.Exitf("pbeam.emitPartitionsNotInTheDataFn.ProcessElement: couldn't encode partition %v: %v", partitionKey, err)
 	}
-	var partitionMap PMap
-	partitionsIter(&partitionMap)
+	var partitionsInDataMap PMap
+	partitionsIter(&partitionsInDataMap)
 	encodedPartitionKey := string(partitionBuf.Bytes())
-	if !partitionMap[encodedPartitionKey] {
-		emit(partitionKey, value)
+	// If partitionsInDataMap is nil, partitionMap is empty, so none of the partitions are in the data, which means we need to emit them.
```

Comment:
nit: `partitionsInDataMap` instead of `partitionMap`

And `which means we need to emit all of them` instead of `which means we need to emit them`